config/logstash.yml

```diff
@@ -281,6 +281,9 @@
 #
 # ------------ Other Settings --------------
 #
+# Run Logstash with superuser (default: ALLOW)
```

Comment:
Whether to allow Logstash to run as superuser (default: ALLOW)

Reply:
Please take a lok at [this PR](https://github.com/elastic/logstash/pull/14089). Thanks.